backend/blueprints/spa_api/spa_api.py

```diff
@@ -514,6 +522,40 @@ def api_handle_error(error: CalculatedError):
     return response
 
 
+@bp.route('/training/create')
+@require_user
+@with_query_params(accepted_query_params=[
+    QueryParam(name="date_start", type_=str, optional=True),
+    QueryParam(name="date_end", type_=str, optional=True)
+])
+def api_create_trainingpack(query_params=None):
+    date_start = None
+    date_end = None
+    if 'date_start' in query_params:
```

Comment:
make sure you check that query_params is not None.

Reply:
Why? It doesn't break when I send no params